CMakeLists.txt

```diff
@@ -1,6 +1,21 @@
-cmake_minimum_required(VERSION 3.23)
+cmake_minimum_required(VERSION 3.13)
 project(coolstory_gram_client)
 
 set(CMAKE_CXX_STANDARD 17)
+set(Boost_USE_STATIC_LIBS OFF)
+set(Boost_USE_MULTITHREADED ON)
+set(Boost_USE_STATIC_RUNTIME OFF)
+find_package(Boost 1.74.0 REQUIRED system)
```

Comment:
Because of `REQUIRED` your `CMakeLists.txt` demands on Boost library in the system. Your PR must add only the folder structure.